MediaWikiKit/MediaWikiKit/MWKSection.m

```diff
@@ -19,20 +21,21 @@ @interface MWKSection ()
 @property (readwrite, strong, nonatomic) MWKTitle* title;
 @property (readwrite, weak, nonatomic) MWKArticle* article;
 
-@property (readwrite, copy, nonatomic) NSNumber* toclevel;      // optional
-@property (readwrite, copy, nonatomic) NSNumber* level;         // optional; string in JSON, but seems to be number-safe?
-@property (readwrite, copy, nonatomic) NSString* line;          // optional; HTML
-@property (readwrite, copy, nonatomic) NSString* number;        // optional; can be "1.2.3"
-@property (readwrite, copy, nonatomic) NSString* index;         // optional; can be "T-3" for transcluded sections
-@property (readwrite, strong, nonatomic) MWKTitle* fromtitle; // optional
-@property (readwrite, copy, nonatomic) NSString* anchor;        // optional
+@property (readwrite, copy, nonatomic, nullable) NSNumber* toclevel;      // optional
```

Comment:
can you please remove the `// optional` comments? maybe even move the other comments in to doc comments above the property?